app/main/db/queries.py

```diff
@@ -81,15 +83,53 @@ def build_fuzzy_search_query(query_string: str, sorting_orders=None):
         ).label("opening_date"),
     )
 
+    if transferring_body_id:
+        query = query.filter(
+            sub_query.c.transferring_body_id == transferring_body_id
+        )
+
     if sorting_orders:
         query = _build_sorting_orders(query, sub_query, sorting_orders)
-    else:
-        query = query.order_by(
-            sub_query.c.transferring_body,
-            sub_query.c.series,
-            sub_query.c.consignment_reference,
-            sub_query.c.file_name,
+    # else:
+    #    query = query.order_by(
+    #        sub_query.c.transferring_body,
+    #        sub_query.c.series,
+    #        sub_query.c.consignment_reference,
+    #        sub_query.c.file_name,
+    #    )
+
+    return query
+
+
+def build_fuzzy_search_summary_query(query_string: str):
+    filter_value = str(f"%{query_string}%").lower()
+
+    fuzzy_filters = or_(
```

Comment:
we could reduce duplication of the shared parts of the summary query and the normal query by extracting those parts into a private function